userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -172,22 +175,70 @@ section. It's meant to be used in combination with the other blocks shortcodes.
 
 ### alert
 
-The **alert** shortcode creates an alert block that can be used to display
-notices or warnings.
+Use the **alert** shortcode to display notices and warnings. The shortcode
+renders a [Bootstrap alert][]. It can be used with Markdown content and contain
+other shortcodes. For example:
 
 ```go-html-template
-{{%/* alert title="Warning" color="warning" */%}}
-This is a warning.
-{{%/* /alert */%}}
+{{%/* alert title="Welcome" */%}} {{%/* param greeting */%}} {{%/* /alert */%}}
```

Comment:
I'm a bit concerned that *just* having this example makes it look like the user needs to use a shortcode to specify the alert message, while (presumably) you can just put text directly in it.

(though obviously we also want to show that they can put shortcodes in it too)

Reply:
I've inlined the greeting.